src/todo/features/main_ui.lua

```diff
@@ -87,11 +91,10 @@ function todo.update_current_task_label(player)
     for _, task in pairs(global.todo.open) do
         if task.assignee == player.name then
             todo.log(serpent.block(task))
-            todo.get_maximize_button(player).caption =
-            {"",
-             {"todo.todo_list"},
-             ": ",
-             task.title
+            todo.get_maximize_button(player).caption = { "",
```

Comment:
I think this is not really readable. Out of the box I would have done something like this:
```
todo.get_maximize_button(player).caption = {
    "", { "todo.todo_list" }, ": ", task.title
}
```

Reply:
Then the line is longer than the recommended length from luacheck :)
But yes.. it's not optimal.